src/MovieSearch.jsx

```diff
@@ -83,8 +108,44 @@ function MovieSearch({ initialRating, onMovieLogged }) {
             <br></br>
 
             {movie.genres}
-          </div>
+          </motion.div>
+        ))}
+        <h3>Movies you may have seen:</h3>
+        {random.map((movie) =>(
+          <motion.div
+          animate={{x:0, scale:1}}
+          whileHover={{ scale: 1.1 }}
+            whileTap={{ scale: 0.9 }}
+            initial={{
+              opacity: 0,
+            }}
+            whileInView={{
+              opacity: 1,
+            }}
+            viewport={{
+              // once: true,
+               margin: "-20px",
+              amount: "all",
+            }}
+          onClick={() => {
+            setMovieSelectedFlag(true);
+            if (initialRating) {
+              logMovieForUser(movie.movieId);
```

Comment:
Is there any indication for the user to know the implication of clicking the movie div?